pegasus/sites.v3/code.org/public/educate/it.md

```diff
@@ -82,12 +83,12 @@ For the very best experience with all Code.org content, we recommend consulting
 
 ## Mobile and Tablet Support Details
 
-All CS Discoveries, CS Principles, and Intro to App Lab courses are not compatible with tablets or mobile devices - a full keyboard and mouse (or touchpad, like on MacBooks) are required. Most tablets are not supported, with the following exceptions:
+All CS Discoveries, CS Principles, CS A, and Intro to App Lab courses are not compatible with tablets or mobile devices - a full keyboard and mouse (or touchpad) are required. Most tablets are not supported, with the following exceptions:
 
 - Microsoft Surface Pro with an external keyboard
 - iPad running iPadOS 13.4 or later with an external keyboard and an external mouse or touchpad
 
-All other content, such as K-5 courses (Computer Science Fundamentals) and Hour of Code activities are compatible on iPads and Chromebooks in addition to all other modern tablets and laptop or desktop computers.
+All other content, such as K-5 courses (Computer Science Fundamentals) and Hour of Code activities are compatible on iPads and other modern tablets and laptop or desktop computers.
```

Comment:
this part is getting a bit harder for me to understand. what do you think about:
```suggestion
All other content, such as K-5 courses (Computer Science Fundamentals) and Hour of Code activities, are compatible on iPads and other modern tablets as well as laptop or desktop computers.
```